### PR TITLE
User can create an expense.

### PR DESCRIPTION
**Pull request by xmichelle**
expense-tracker-1

### CODE REVIEW

**Comment by xmichelle** [comment]
Just realized that my form doesn't reset after submitting it. I'll commit the main.js file with the new code.

File: expenses.sql [hunk 1]
```diff
@@ -0,0 +1,14 @@
+DROP TABLE IF EXISTS expenditures;
+
+CREATE TABLE expenditures (
+  id                 serial,
+  user_id            integer,
+  category_id        integer,
+  item               text,
+  transaction_date   numeric,
```
Comment (victormejia) [inline expenses.sql]: numeric may be overkill for this, maybe something like `bigint`: https://www.postgresql.org/docs/9.1/static/datatype-numeric.html

File: expenses.sql [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  amount             numeric(1000, 2)
+);
+
+/*SELECT TO_CHAR(date, 'Mon dd, yyyy');*/
+
+/*SELECT * FROM expenditures;*/
```
Comment (victormejia) [inline expenses.sql]: delete uncommented code

File: index.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+})
+
+app.post('/expenditures', (req, res) => {
+  const expenseData = req.body
+  knex
+    .insert(expenseData)
+    .into('expenditures')
+    .returning('*')
+    .then((data) => {
+      console.log(data)
```
Comment (victormejia) [inline index.js]: remove console logging

File: public/main.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+//     console.log(jsonData)
+//     // jsonData.forEach(data => {
+//     //   const $div = document.createElement('div')
+//     //   $div.textContent = data
+//     //   document.body.appendChild($div)
+//     // })
+//   })
+//   .catch(err => {
+//     console.log(err)
+//   })
```
Comment (victormejia) [inline public/main.js]: let's delete all this commented code

File: public/main.js [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+  fetch('/expenditures', {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json'
+    },
+    body: JSON.stringify(newExpense)
+  })
+    .then(res => res.json())
+    .then(data => console.log(data)) // to view history of expenses, create a .then function that appends info to the document.
```
Comment (victormejia) [inline public/main.js]: indentation is off here, and delete the comment

**Review by victormejia — CHANGES_REQUESTED** [review]
Awesome, awesome job here! Minor changes required.

File: public/main.js [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+
+  fetch('/expenditures', {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json'
+    },
+    body: JSON.stringify(newExpense)
+  })
+      .then(res => res.json())
+      .then(data => console.log(data))
```
Comment (victormejia) [inline public/main.js]: let's fix the indentation here ( only 2 spaces for the .then)

**Review by victormejia — APPROVED** [review]
💥

**Pull request merged**